feconf.py

```diff
@@ -1275,7 +1274,9 @@ def get_empty_ratings():
     constants.ACTIVITY_STATUS_PRIVATE, constants.ACTIVITY_STATUS_PUBLIC]
 
 # Commands allowed in CollectionRightsChange and ExplorationRightsChange.
-COMMON_RIGHTS_ALLOWED_COMMANDS = [{
+COMMON_RIGHTS_ALLOWED_COMMANDS: List[
+    Dict[Text, Union[Text, List[Text], Dict[Text, Union[Text, List[Text]]]]]
```

Comment:
Can we introduce a type variable for the commands? The structure seems pretty similar.

Reply:
Made a new variable `TYPE_COMMAND = Dict[Text, Union[Text, List[Text], Dict[Text, Union[Text, List[Text]]]]`and used it in feconf.py. PTAL.
Thanks!